### DISCUSSION TITLE
Gridpoints forecast not working

### DISCUSSION THREAD

**Discussion by swbrains** [weather-gov/api#533] [Q&A]
When I call the following API endpoint for these grid points, it gives an error 503 [Service Unavailable], and has for several days:

https://api.weather.gov/gridpoints/FWD/101,110/forecast

Earlier in the week, it returned 500 [Internal Server Error] a number of times, and seems to have done it many times over the past month according to my error logs.

**Comment by JJKraw** [comment]
A known problem. A couple of workarounds you can try:

1) Retry. I got it the second time around.

2) Use preview-api.weather.gov.  I've seen 50x errors there also but not as much (could be just dumb luck).
